packer/linux_debian.pkr.hcl

```diff
@@ -6,37 +6,46 @@ locals {
   image_identity = "${var.image_name}-${var.image_date}"
 
   debian_gcp_images = [
+    {
+      task_name = "bookworm"
+      zone = "us-west1-a"
+      machine = "t2d-standard-2"
+      source_image_family = "debian-12"
+    },
     {
       task_name = "bullseye"
       zone = "us-west1-a"
       machine = "t2d-standard-2"
+      source_image_family = "debian-11"
     },
     {
       task_name = "sid"
       zone = "us-west1-a"
       machine = "t2d-standard-2"
+      source_image_family = "debian-11"
```

Comment:
Should probably build sid* based on debian-12...

Reply:
I didn't change them because I thought you may not want to upgrade them right now, done.

Reply:
When debian-12 is used, all sid* images are failing with:

```
 The following packages have unmet dependencies:
 libkrb5-26t64-heimdal : Breaks: libkrb5-26-heimdal (< 7.8.git20221117.28daf24+dfsg-5)
 E: Unable to correct problems, you have held broken packages.
```

Reply:
I think the problem is just that for bookworm a sources list for bookworm survives in `/etc/apt/sources.list.d/debian.sources`, which leads to the globs matching both bookworm and unstable packages. Leading to the error.

You're right that it might be better to change this separately.